code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -247,6 +247,14 @@
 	if(new_ringtone && (new_ringtone != MESSENGER_RINGTONE_DEFAULT))
 		update_ringtone(new_ringtone)
 
+	var/datum/computer_file/program/themeify/theme_app = locate() in stored_files
+	if(theme_app)
+		var/list/unlocked_themes = owner_client.get_award_status(/datum/award/score/progress/pda_themes)
```

Comment:
Likely due to the `.Find()` runtime from the previous review comment, no achievements are currently unlocked as well.